docs/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -5,10 +5,18 @@ sidebar_label: "Configuration variables"
 description: "Learn more about core configuration, component configuration, database configuration, and feature flags."
 ---
 
-As a Spring Boot application, Identity supports any standard
-[Spring configuration](https://docs.spring.io/spring-boot/reference/features/external-config.html) method.
+Identity can be configured using environment variables, configuration parameters, or a combination of both. When configuring your Identity setup, keep in mind the following:
 
-## Core configuration
+- If both configuration files and environment variables are present, environment variables overwrite settings in configuration files.
+- The existing configuration is applied at startup, and changes made to the configuration will not be applied at runtime.
+
+For more information on Self-Managed configuration options, see [configuring components](/self-managed/operational-guides/application-configs.md).
+
+## Environment variables
+
+:::note
+Underscores in environment variables correspond to configuration file key levels.
```

Comment:
Maybe this is true for all configuration options in identity, but I don't think it holds true for other components. I think it would be better to use `Generally,...` at the start